arch/models.py

```diff
@@ -46,6 +49,41 @@ def get_absolute_url(self):
     def get_statement(self) -> Optional[str]:
         return get_disk_statement_from_puid(self.puid)
 
+    @property
+    def niceness(self) -> float:
+        total: int = 0
+
+        votes = self.vote_set.all()
+
+        if len(votes) > 0:
+            for vote in votes:
+                total += vote.niceness
+
+            total /= len(votes)
```

Comment:
i would rather just `return round(total / len(votes), 2)` here, shorter and doesn't cause a type clash

Reply:
aren't you dividing by len(votes) twice now?

seems like you should move the subsequent `        return round(total / len(votes), 2)` here and just delete this line